newIDE/app/src/AssetStore/PrivateAssets/PrivateAssetPackInformationDialog.js

```diff
@@ -278,6 +279,21 @@ const PrivateAssetPackDialog = ({
                                 <Trans>Redistribution &amp; reselling</Trans>
                               </Text>
                             </LineStackLayout>
+                            <Line noMargin>
+                              <Text>
+                                <Trans>See details</Trans>{' '}
+                                <Link
+                                  onClick={() =>
+                                    Window.openExternalURL(
+                                      'https://gdevelop.io/page/asset-store-license-agreement'
+                                    )
+                                  }
+                                  href="https://gdevelop.io/page/asset-store-license-agreement"
+                                >
+                                  <Trans>here</Trans>
```

Comment:
A bit hard to translate ("Voir détail" "ici" even in French is not super great), I think you can put the link on the whole text "See details here."?

Reply:
Good idea, I'll change it